docs/docsite/rst/plugins/lookup.rst

```diff
@@ -12,6 +12,9 @@ Lookup plugins are an Ansible-specific extension to the Jinja2 templating langua
 .. note::
    - Lookups are executed with a working directory relative to the role or play,
      as opposed to local tasks, which are executed relative the executed script.
+     The lookup plugins such as ``file``, ``fileglob``, ``template``, and ``filetree`` are exceptions to this as they have a list of path candidates to search files in.
```

Comment:
they are not exceptions, they still execute relative to role or play

what they do differently is handle 'relative paths' in their search, but not in their execution